app.py

```diff
@@ -0,0 +1,75 @@
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from flask import Flask, render_template, request, redirect, url_for
+from pymongo import MongoClient
+
+import datetime
+import json
+import os
+import sys
+
+client = MongoClient(os.environ['MONGODB_URI'])
+db = os.environ['MONGODB_URI'].split('/')[-1]
+app = Flask(__name__, static_url_path='')
+
+
+def get_5_last_documents():
+    return list(client[db].coll.find().sort("ts", -1).limit(5))
+
+
+@app.route('/add_document', methods=['POST'])
+def add_document():
+    error_msg = None
+
+    try:
+        document = json.loads(request.form['document'])
+        document["ts"] = str(datetime.datetime.now())
+    except Exception as e:
+        error_msg = "Converting to JSON failed: {}".format(e)
+
+    if not error_msg:
+        try:
+            resp = client[db].coll.insert(document)
+        except Exception as e:
+            error_msg = "Inserting into MongoDB failed: {}".format(e)
+
+    results = get_5_last_documents()
```

Comment:
We don't seem to be doing anything with `results` here? E.g. a linter would error about this. Not sure if that represents an oversight (did we want to display part of it to the user on success?) or if it's just copypasta that can get axed.

Reply:
oversight (and a now remedied lack of linter)